tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -20,6 +23,37 @@ def __init__(self):
         super(GlusterIntegrationSdsSyncStateThread, self).__init__()
         self._complete = gevent.event.Event()
 
+    def _emit_event(self, resource, curr_value, msg):
+        alert = {}
+        alert['source'] = 'tendrl-gluster-integration'
+        alert['pid'] = os.getpid()
+        alert['timestamp'] = tendrl_now().isoformat()
+        alert['alert_type'] = 'status'
+        severity = "info"
+        if curr_value == "Stopped":
+            severity = "critical"
+        alert['severity'] = severity
+        alert['resource'] = resource
```

Comment:
Resource name should be something like `volume_status` and the same should be populated as `handles` field of a handler which needs to added under `tendrl-alerting` module as `handlers/cluster/volume_status_handler.py`

You can refer https://github.com/Tendrl/alerting/pull/78 for more details.

If required a separate handler to be written for brick status changes.